qhana_plugin_runner/db/models/mutable_json.py

```diff
@@ -0,0 +1,300 @@
+# Copyright (c) 2014, Elmer de Looff <elmer.delooff@gmail.com>
```

Comment:
Is it possible to install this from PyPi or from the github repo as a dependency? I will need to think about how to best include this sourcecode (and its licence).

Reply:
We do need to overwrite NestedMutable in order to accept primitive types without nesting. It should be possible though to import NestedMutableDict and NestedMutableList (which is basically all that is required from the lib) and add it as a dependency. That would reduce the file by about two thirds.